.travis/travis-deploy.sh

```diff
@@ -18,6 +18,7 @@ if [ "$TRAVIS_PULL_REQUEST" = "false" ] && [ "$TRAVIS_BRANCH" = "master" ]; then
     # push jenkins plugin snapshot to https://repo.jenkins-ci.org/snapshots/fr/inria/repairnator
     cd repairnator-jenkins-plugin
     mvn -q deploy -DskipTests
+    cd ..
```

Comment:
I'm not sure, because nothing is executed after this line.

Reply:
I am proposing this due to the error output description ``unable to prepare context: path "docker-images/pipeline-dockerimage" not found` on travis . seemingly fails at [this line](https://github.com/eclipse/repairnator/blob/1a53241dbd7d54b461be7f032258b0bead993d56/.travis/travis-deploy.sh#L34)

Reply:
deployment of the plugin worked only the docker stage fails , https://repo.jenkins-ci.org/snapshots/fr/inria/repairnator/repairnator-jenkins-plugin/2.0-SNAPSHOT/ ,